pkg/webservice/webservice.go

```diff
@@ -41,26 +42,29 @@ type WebService struct {
 	httpServer *http.Server
 }
 
-func newRouter() *mux.Router {
-	router := mux.NewRouter().StrictSlash(true)
+func newRouter() *httprouter.Router {
+	router := httprouter.New()
 	for _, webRoute := range webRoutes {
 		handler := loggingHandler(webRoute.HandlerFunc, webRoute.Name)
-		router.
-			Methods(webRoute.Method).
-			Path(webRoute.Pattern).
-			Name(webRoute.Name).
-			Handler(handler)
+		switch webRoute.Method {
+		case "GET":
+			router.HandlerFunc(http.MethodGet, webRoute.Pattern, handler)
+		case "PUT":
+			router.HandlerFunc(http.MethodPut, webRoute.Pattern, handler)
+		case "POST":
+			router.HandlerFunc(http.MethodPost, webRoute.Pattern, handler)
+		}
```

Comment:
Unneeded switch, http.Method* are simple strings use:
```
router.Handle(webroute.Method, webroute.Pattern, handler)
```

Reply:
My bad, this is more reasonable! Thank you @wilfred-s !